sandbox/Sandbox.jsx

```diff
@@ -50,6 +50,8 @@ export default class Sandbox extends React.Component {
         };
     }
 
+    onClickCanvas = () => console.info(`Clicked the canvas`);
```

Comment:
Please rename this to `onClickGraph`, people may mistakenly think that under the hood canvas is being used when the full `react-d3-graph` implementation is based on `SVG` only.